### PR TITLE
Merge on Eternal Palace raid day release!

### PR DESCRIPTION
**Pull request by yajinni**
Changed character search default to Eternal Palace

### CODE REVIEW

File: src/CHANGELOG.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ import SpellLink from 'common/SpellLink';
 import Contributor from 'interface/contributor/Button';
 
 export default [
+  {
+    date: new Date('2019-06-19'),
```
Comment (MartijnHols) [inline src/CHANGELOG.js]: Might want to set this to the release date so it's actually visible on release :)

**Comment by MartijnHols** [comment]
thanks!

File: src/CHANGELOG.js (hunk 1 above)
Comment (yajinni) [reply src/CHANGELOG.js]: fair enough lol

**Comment by Sharrq** [comment]
Just need to fix the conflict in the changelog.

**Pull request merged**